homeassistant/components/xiaomi/camera.py

```diff
@@ -57,12 +58,14 @@ class XiaomiCamera(Camera):
     def __init__(self, hass, config):
         """Initialize."""
         super().__init__()
+        self.hass = hass
```

Comment:
Don't set `self.hass`. It will be set on the entity when it has been added to home assistant.

Reply:
so, will it work by removing it?.

Reply:
Yes.